### DISCUSSION TITLE
TweakScale giving error "Houston we have a problem!" help please.

### DISCUSSION THREAD

**Discussion by johnycraft** [TweakScale/TweakScale#266] [Support] [answered]
@Lisias

Hey sorry if i am being stupid but i am having an issue with this mod saying houston we have a problem! Unfortunatly tweakscale didn't found needed dlls. i have been having this issue for awhile now i have tried uninstalling all my mods and reinstalling and it still happends here is an image of the error thing that shows up on game load. and also an image of my game data file any help would be great thank you sorry for being a pain but this is starting to get annoying.

![KSP TweakScale Error](https://user-images.githubusercontent.com/86728726/197932866-ed924843-81aa-47b3-a940-980cf3b245a8.PNG)

![KSP Game Data 1](https://user-images.githubusercontent.com/86728726/197932889-994b9717-9529-46d0-be79-ea74c4f26d68.PNG)

**Comment by Lisias** [comment]
Hi, @johnycraft !

You are being bitten by a nasty KSP bug on a thingy called Assembly Loader/Resolver (ALR for shorts) . In a nutshell, it's a component that is responsible for loading DLLs on the whole system - and it's broken on KSP: when something fails to be loaded, **everything else** fails too from that point, and TweakScale is the only one that yells when this happens (everybody else just try to keep going, what most of the time corrupts the savegames beyound hope).

In oder to under to understand what's happening, I need your KSP.log . You will find it the same place where the ksp.exe file is.

Zip it and drop it on a post here, and I will check it to see who is triggering the ALR problem. Keep in mind that whatever I will tell you is the problem, in essence is only the **trigger** of the problem, and not the culprit itself!

Cheers!

(mention my nome using @Lisias  on the post, so github will notify me about the post for sure!)

**Comment by johnycraft** [comment]
@Lisias 

Thanks for the information here is my log i updated your mod and it does it more so yeah if you are able to tell me what mod is triggering it that would be great thank you.

[KSP.log](https://github.com/net-lisias-ksp/TweakScale/files/9998823/KSP.log)

**Comment by Lisias** [comment, marked as answer]
@johnycraft please try the latest 2.4.6.18 version, I just published it everywhere.

I had borked on the release you installed, a merge error passed through and by a plain lack of luck, by 1.12.4 testbed was contaminated in a way that such mistake would not be detected.

Let me know if it will fix things for you!

**Reply by Lisias** [reply]
Marking it as answer, as the user didn't came back for feedback - so I'm assuming the thing is working.
